src/components/ResourceList/ResourceList.tsx

```diff
@@ -533,6 +537,7 @@ export class ResourceList extends React.Component<CombinedProps, State> {
     const resourceListClassName = classNames(
       styles.ResourceList,
       loading && styles.disabledPointerEvents,
+      selectMode && styles.disableTextSelection,
```

Comment:
what was happening without this?

Reply:
You'll select text when multi-selecting, really bad experience 😬